src/parsing/common/page.ts

```diff
@@ -5,8 +5,8 @@ import { Prediction } from "./prediction";
 import { StringDict } from "./stringDict";
 
 export class Page<T extends Prediction> {
-  id: number | undefined;
-  orientation: OrientationField | undefined;
+  id?: number;
```

Comment:
when is the page ID missing? It should always be present. Possible bug in API.

Reply:
Previous tests ignored page n° so I assumed it would be the same with the new ones. The pages do indeed always have an ID now, though.